python/lsst/source/injection/bin/make_injection_pipeline.py

```diff
@@ -120,6 +120,32 @@ def build_argparser():
         help="Config override for a task, in the format 'label:key=value'.",
         action="append",
     )
+    parser.add_argument(
+        "-ip",
+        "--input-pipelines",
```

Comment:
Standard practice is for the single `-` dash arguments to be a single character. How about renaming this 
```
        "-a",
        "--additional-pipelines",
```
?